### ISSUE TITLE
How to set es6 in VScode?

### ISSUE THREAD

**Issue by rowild** [vuejs/vue-cli#1334]
I am testing the vue-cli with the webpack package.

The eslinter does report that e.g. import is only available in es6 and that i should use `"esversion": 6` But I do not know where.
Putting it in the `module.exports` section of `.eslintrs.js`does not change anything, and the link given in a comment there does not provide any solution.

Any hints?

**Comment by rowild** [comment]
So where are the help forums?

**Comment by kayandrae07** [comment]
https://forum.vuejs.org/

You can recreate the issue, just make sure to follow the guidelines defined at https://new-issue.vuejs.org/?repo=vuejs/vue-cli

**Comment by LinusBorg** [comment]
> You can recreate the issue, 

No, he may not. he wants support with using eslint, which we don't provide in github issues.

Help can be found [here ](forum.vuejs.org)or [here](chat.vuejs.org), which are linked to from https://new-issue.vuejs.org/?repo=vuejs/vue-cli, but which he obviously didn't care to read.
